Dockerfile-autogen-models

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:22.04
+
+RUN apt update && \
+    apt install -y --no-install-recommends \
+    curl \
+    default-jdk && \
+    apt clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# Ref.: https://nodejs.org/en/download/package-manager
+ENV NVM_DIR=/root/.nvm
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.0/install.sh | bash && \
+    . "$NVM_DIR/nvm.sh" && \
+    nvm install 22.11.0 && \
+    nvm alias default 22.11.0 && \
+    nvm use default
+
+ENV PATH=$NVM_DIR/versions/node/v22.11.0/bin:$PATH
+
+WORKDIR /app
+
+# Note: Not working...
+# RUN npm install -g @openapitools/openapi-generator-cli
```

Comment:
Why is it not working? :thinking: Did you try using the official image `openapitools/openapi-generator-cli`?

Then maybe you could copy in the model build script and create an entry point for it, so that we can do something like?

```
docker run -it -v $(pwd):/app ga4gh-sdk-autogen-models bash <OPENAPI_SPEC_PATH> <API_NAME> <DESTINATION_DIR>
```

Reply:
@suecharo do you plan to switch within this PR?

Reply:
@pavelnikonorov @uniqueg 
I fixed it within the same PR.
I also tried to use the official image, `openapitools/openapi-generator-cli`, but it assumes execution with `java <something>.jar` , so I decided to keep the Dockerfile after all.